deployment/config.js

```diff
@@ -45,9 +45,9 @@ module.exports = {
 			type: 'object',
 			patternProperties: {
 				'.+': {
-					type: 'object',
-					required: ['max', 'min'],
-					properties: {
+					'type': 'object',
```

Comment:
Weirdly but eslint complains with quote-props error unless I wrap three following keys into quotes:
![image](https://user-images.githubusercontent.com/182853/48112585-80356e80-e270-11e8-8ecc-870ca800bf1c.png)

Reply:
It's because once you quote one property, it wants you to quote _all_ the properties in that object "for consistency"

Reply:
I see. But since `if` is a reserved word, eslint requires it to be quoted.